app/models/repo.rb

```diff
@@ -1,10 +1,14 @@
 class Repo < ActiveRecord::Base
   has_and_belongs_to_many :cohorts
-  has_many :submissions
+  has_many :submissions, dependent: :destroy
   validates :repo_url, presence: true
-  
+
   BASE_GITHUB_URL = "http://github.com/"
 
+  def self.default_sort
```

Comment:
Have you considered using a default scope for this sort behavior?
```ruby
default_scope { order(created_at: :desc) }
```

This would also allow us to avoid referencing `default_sort` in the controller actions (or wherever we're using `Repo.all`).